frontend/src/router/index.ts

```diff
@@ -2,8 +2,6 @@ import Vue from "vue";
 import VueRouter, { RouteConfig } from "vue-router";
 import store from "../store";
 
-import auth from "./auth";
-
```

Comment:
I've been wondering if this was actually in use...
(out of scope 😛) 

But we should create an issue to delete `auth.ts` if it doesn't do anything

Reply:
I created  #115.

Reply:
Yes, I created #115.